hudi-common/src/main/java/org/apache/hudi/common/model/ClusteringGroupInfo.java

```diff
@@ -67,6 +75,10 @@ public void setNumOutputGroups(final int numOutputGroups) {
     this.numOutputGroups = numOutputGroups;
   }
 
+  public Map<String, String> getExtraMetadata() {
```

Comment:
Return `Option` to avoid null value returned.

Reply:
> Return `Option` to avoid null value returned.

done~